src/MonitoringFunctions/Properties/serviceDependencies.local.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "storage1": {
+      "resourceId": "/subscriptions/[parameters('subscriptionId')]/resourceGroups/[parameters('resourceGroupName')]/providers/Microsoft.Storage/storageAccounts/monitoringfuncsstorage",
+      "type": "storage.azure",
+      "connectionId": "AzureWebJobsStorage"
+    }
+  }
+}
```

Comment:
white space

Reply:
Must be placed by github. There is no whitespace in the file. Besides, these files are managed by visual studio. Why should we concern ourselves with white spaces?